storage-account-ld-location.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+  product                   = "ldlocation"
+  ld_location_account_name  = "${local.product}${var.env}"
+  container_name            = "lrd-ref-data"
```

Comment:
can you plc change stg account name to rdlocationaat?

Reply:
Change completed.